developer-model-service/docs/access.md

```diff
@@ -0,0 +1,139 @@
+# Access & Usage Information
+
+## Basic Information
+
+### Model Server URL
+
+The model server can be accessed [here](https://model-service.apps.domain.com/).
+
+## Authentication
+
+In order to gain access to the model server you will need to sign in with your SSO credentials by selecting Google auth. Once you have signed in you are able to generate a token by navigating to **`Apps and API Keys`** on the top ribbon and hitting **`Create new Application`**:
+
+![Generation Example](../images/generation-example.png)
+
+Once a token has been generated, you will be provided with the API server URL.
+
+**Note:** If you are unable to access the model server via SSO you need to reach out to a platform administrator.
+
+## API Schema
+<!--
+The name of the api, model-service-api, is grabbed from the name field in the ai-catalog.yaml metadata for the api.
+
+TechDocs read this as a URL starting from where the docs are located, so we can use ../ behaviour to navigate the TechDocs to reference other resources/components/apis
+-->
+The API Schema is available [here](../../../../../api/model-service-api/definition).
```

Comment:
Since moving `access.md` from `/code` it doesn't need to `../` an extra level higher, this would most likely work:
```suggestion
The API Schema is available [here](../../../../api/model-service-api/definition).
```

Reply:
Switching to absolute paths seemed to fix this